apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -118,7 +118,7 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
     },
     hooks: {
         validateInput: ({ resolvedData, existingItem, context, addValidationError }) => {
-            if (!hasRequestFields(['dv', 'sender'], resolvedData, context, addValidationError)) return
+            if (!hasDvAndSenderFields( resolvedData, context, addValidationError)) return
```

Comment:
Am I right, by saying that right now we would always need to set `hasDvAndSenderFields` in validation for our feature models? 

Maybe add it to codegen?

Reply:
It will be nice!